app/src/main/res/layout/username_password_layout.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    tools:showIn="@layout/activity_main">
+
+    <EditText
```

Comment:
To Avoid many style of this Edit text, should create style of email box for 
- text color
- maxlines
- ...
except the layout style